subgraph.yaml

```
@@ -150,3 +204,30 @@ dataSources:
         - event: Settled(uint256,uint256,bool,indexed address)
           handler: handleSettled
       file: ./src/market.ts
+
+  - kind: ethereum/contract
+    name: HL Mock DIA 3
+    network: goerli
+    source:
+      address: "0x5B8C51684a3084a58a68b70EbED7B259D2BCab23"
+      abi: Market
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Protocol
+        - Bet
+      abis:
+        - name: Market
+          file: ./abis/Market.json
+      eventHandlers:
+        - event: Claimed(indexed address,uint256)
+          handler: handleClaimed
+        - event: MarketOwnershipTransferred(indexed address,indexed address)
```

Not sure where you're getting that event name. It should be `OwnershipTransferred` from OZ `Ownable`, unless I've missed something?
```suggestion
        - event: OwnershipTransferred(indexed address,indexed address)
```

---

updating the abi resolved this